frontend/src/pages/ProjectDetails/index.jsx

```diff
@@ -21,10 +21,11 @@ import {
 const ProjectDetails = () => {
   const data = {
```

Comment:
We can't merge with dummy data. This needs to be connected to the backend since the appropriate api endpoint exists.

Reply:
Backend Connections are completed, it is ready to be merged.